include/emulator/flash.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _FLASH_H
+#define _FLASH_H
+
+#include "emulator/xlObject.h"
+#include "revolution/types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// __anon_0x7428F
```

Comment:
I don't think these `__anon` comments are useful here (they're probably not in oot-gc either anymore but that's another story)